pandas/io/sas/api.py

```diff
@@ -0,0 +1,3 @@
+from pandas.io.sas.sas7bdat import SAS7BDATReader
```

Comment:
why are we exposing the actual reader objects? would prefer you just expose ``read_sas``, then import the readers when needed (so everything isn't imported at right away).